### PR TITLE
Update issue_template.md

### PR DESCRIPTION
**Pull request by kannankvs**
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] master

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -50,6 +50,11 @@
 ```
 (paste your output here)
 ```
+#### Output of `show tech support`:
```
Comment (jleveque) [inline .github/issue_template.md:53]: Should read `show techsupport` (no space)

**Review by jleveque — CHANGES_REQUESTED** [review]

File: .github/issue_template.md (hunk 1 above)
Comment (kannankvs) [reply .github/issue_template.md]: addressed

**Comment by jleveque** [comment]
@lguohan: Azure Pipelines doesn't want to run on this PR. It doesn't really matter, since this is not a code change, but I'm not sure if it's expected to block the PR.

Can you please review, approve and merge this using your admin permissions?

**Review by jleveque — APPROVED** [review]

**Pull request merged**